tests/dashboard/guest-management.test.js

```diff
@@ -3,6 +3,7 @@ const common = require('../common'),
 
 describe('Guest Management', () => {
   before(() => common.login(browser, 'admin', 'admin', 'local'));
+  after(() => common.logout());
   it('should navigate to Guest Management', () =>
     common.clickSidebarTab(browser, 'Guest Management'));
 });
```

Comment:
This one will have to be revisited once you have the full lifecycle testing - as successfully testing this page requires either a laundry list of prerequisites (template, pool, and potentially realm).

In addition, some of the test cases might require injection of canned records to the DB. for example, the options that are enabled in a guest's dropdown is related to it's "status", for example, soft power command (reboot/shutdown) should only be enabled for 'Ready', 'Activated' and, 'Deactivated' states, undefine only enabled for 'Shut Off' state, etc.